tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -0,0 +1,105 @@
+import { sortBy } from 'common/collections';
+import { useBackend } from "../backend";
+import { Window } from "../layouts";
+import { NanoMap, Box, Table, Tooltip, Icon, Button } from "../components";
+
+export const CrewMonitor = (props, context) => {
+  const { act, data } = useBackend(context);
+  const crew = sortBy(
+    crewmember => crewmember.name,
+  )(data.crewmembers || []);
+  return (
+    <Window>
+      <Window.Content>
+        {crew.map(crewmember => (
+          (crewmember.sensor_type === 3 ? (
+            <Box
+              name={crewmember.sensor_type === 3 ? "circle" : "none"}
+              position="absolute"
+              className="NanoMap_Marker"
+              top={(((255 - crewmember.y) * 2) + 2) + 'px'}
+              left={((crewmember.x * 2) + 2) + 'px'}
+              tooltip={crewmember.name}>
+              <Icon
+                name="circle"
+                color={crewmember.dead ? 'red' : 'green'}
+                size={0.5}
+              />
+              <Tooltip content={crewmember.name} />
+            </Box>
+          ) : (null))
+        ))}
+        <NanoMap />
+        <div className="NanoMap_Content_Offset">
+          <Box bold m={2}>
+            <Table>
+              <Table.Row header>
+                <Table.Cell>
+                  Name
+                </Table.Cell>
+                <Table.Cell>
+                  Status
+                </Table.Cell>
+                <Table.Cell>
+                  Location
+                </Table.Cell>
+              </Table.Row>
+              {crew.map(crewmember => (
+                <Table.Row key={crewmember.name}>
+                  <td>
```

Comment:
Any reason why you're using `<td>` in stead of `<Table.Cell>` in this section?

Reply:
Does my brain being in autopilot from 8 hours of webdesign count?